app/Http/Controllers/InvoicesController.php

```
@@ -92,7 +89,7 @@ public function show(Invoice $invoice)
             ->withPaymentSources(PaymentSource::values())
             ->withAmountDue($amountDue)
             ->withSource($invoice->source)
-            ->withCompanyName(Setting::first()->company);
+            ->withCompanyName(Setting::cached()->company);
```

PHPStan (2.2.5)

[warning] Cannot access property $company on App\Models\Setting|null.


(property.nonObject)